src/server/app.py

```diff
@@ -1,15 +1,20 @@
 import json
 
-from flask import Flask, request, Response
+from flask import Flask, request, Response, send_from_directory
 from flask_cors import CORS
 
 import src.server.cea_608_encoder.byte_pair_generator as encoder
 
 
-app = Flask(__name__)
+app = Flask(__name__, static_folder='../client/')
```

Comment:
Does the page load with the static folder argument removed?

Reply:
It still works, removed!